index.css

```diff
@@ -0,0 +1,137 @@
+
+.topnav {
+  display: flex;
+  justify-content: space-between;
+  color: white;
+  background-color: #3e3c3c;
+  height: 92px;
+}
+
+.topnav #myLinks {
+  display: none;
+}
+
+a {
+  float: left;
+  color: white;
+  padding: 14px 16px;
+  text-decoration: none;
+  font-size: 17px;
+}
+
+.active {
+  position: relative;
+  left: 30px;
+  top: 22px;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 24px;
+}
+
+a .active {
+  width: 77px;
+  height: 24px;
+  font-family: "Roboto", sans-serif;
+}
+
+.menor {
+  width: 54px;
+  height: 16px;
+  left: 0;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 14px;
+  line-height: 16px;
+  text-align: center;
+  color: #fff;
+  padding-left: 40px;
+}
+
+nav {
+  display: flex;
+  align-items: center;
+}
+
+.barra {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.roger {
+  height: 380px;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 120px 24px;
+  position: relative;
+  background: #1c1a19 url(pictures/Group.png) 100% 110% no-repeat;
+}
+
+.pad {
+  position: relative;
+  left: 20px;
+  color: #ff6b00;
+  width: 327px;
+  height: 124px;
+  bottom: 50px;
+  font-family: "Crete Round", serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 37px;
+  letter-spacing: 0.37px;
+}
+
+.k {
+  position: relative;
+  bottom: 50px;
+}
+
+.supporttest {
+  color: white;
+  width: 300px;
+  height: 27px;
+  position: relative;
+  left: 20px;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  line-height: 20px;
+}
+
+.theme1 {
+  color: white;
+  width: 280px;
+  height: 123px;
+  left: 24px;
+}
```

Comment:
[Optional ] I see that you used position relative and position absolute in your code,
position relative count the position from the original  html overflow while position absolute count the position from the point (0,0) in the screen this causes problems to the body overflow because the width of the childelement will never increase the width of the parent element so Please avoid using these position and try to style your code using only flex and width and margins so you can solve the overflow problem
![image](https://user-images.githubusercontent.com/74744787/158717137-7b7927ba-9b99-4467-b769-2bcb5ad27518.png)